dev/tools/grunt/configs/local-themes.js

```diff
@@ -0,0 +1,64 @@
+/**
```

Comment:
Config files related to NPM, GRUNT, COMPOSER, and others should not be pushed in the commit remove this file from here